doc/development/developing-sql-dialect/implementing_mandatory_sql_dialect_classes.md

```diff
@@ -42,18 +42,18 @@ And we also need two corresponding test classes:
    } 
    ```
 
-1. **Add a constructor** that takes a [JDBC database connection](https://docs.oracle.com/javase/8/docs/api/java/sql/Connection.html) and user properties as parameters. 
+1. **Add a constructor** that takes a [JDBC connection](https://docs.oracle.com/javase/8/docs/api/java/sql/Connection.html) Factory and user properties as parameters. 
     Your IDE could also generate it for you.
```

Comment:
```suggestion
    Or let your IDE generate it for you. You might wonder, why you get a factory for a connection instead of a connection. This allows your dialect to decide if and when you really need a connection to the remote data source. Depending on that source, establishing a connection can be costly, so having full control over connection creation can safe resources and execution time.
```